httpserver/http_connection.cpp

```diff
@@ -480,6 +483,34 @@ void connection_t::process_retrieve_all() {
     response_.result(http::status::ok);
 }
 
+bool connection_t::check_correct_swarm(std::string pub_key) {
```

Comment:
const ref ?
Usually you would take by value only if you _want_ to take a copy because you don't want to mutate the original object.

Reply:
Roger, honestly don't think I have thought about const w/r/t arguments at all lol but will try to start doing it consciously

Reply:
Haha, once you get drilled, there's no going back.You can see the signature as a business card that helps figuring out quickly what the intentions of the functions are:
* `foo(std::string str)` will take a copy of your string, probably to mutate it, leaving your original variable untouched.
* `foo(std::string& str)`will take your string and mutate it directly.
* `foo(const std::string& str)`will take your string but I will only read from it.